MAUI/Maps/migration.md

```diff
@@ -349,6 +351,302 @@ The following code example explains how to intialize the layer and customize the
+   </td>
+   <td>
+      {{'[*Center]()'| markdownify }} 
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[CanCacheTiles](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_CanCacheTiles)'| markdownify }}
+   </td>
+   <td>
+      {{'[CanCacheTiles]()'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[Radius](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_Radius)'| markdownify }} 
+   </td>
+   <td>
+      {{'[Radius]()'| markdownify }} 
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[DistanceType](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_DistanceType)'| markdownify }} 
+   </td>
+   <td>
+      {{'[DistanceType]()'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[LatLngBounds](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_LatLngBounds)'| markdownify }} 
+   </td>
+   <td>
+      {{'[MapLatLngBounds]()'| markdownify }} 
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[Markers](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.MapLayer.html#Syncfusion_SfMaps_XForms_MapLayer_Markers)'| markdownify }}
+   </td>
+   <td>
+      {{'[Markers](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLayer.html#Syncfusion_Maui_Maps_MapLayer_Markers)'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[MarkerTemplate](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.MapLayer.html#Syncfusion_SfMaps_XForms_MapLayer_MarkerTemplate)'| markdownify }}
+   </td>
+   <td>
+      {{'[MarkerTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLayer.html#Syncfusion_Maui_Maps_MapLayer_MarkerTemplate)'| markdownify }}
+   </td>
+</tr>
+</table>
+
+<b>Methods</b>
+
+<table>
+<tr>
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+   <td>
+     {{'[DeleteTilesFromCache()](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_DeleteTilesFromCache)'| markdownify }} 
+   </td>
+   <td>
+      {{'[DeleteTilesFromCache()]()'| markdownify }} 
+   </td>
+</tr>
+</table>
+
+<b>Events</b>
+
+<table>
+<tr>
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+   <td>
+     {{'[ZoomLevelChanging](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_ZoomLevelChanging)'| markdownify }} 
+   </td>
+   <td>
+      {{'[ZoomLevelChanging]()'| markdownify }} 
+   </td>
+</tr>
+</table>
+
+N> All the (*) marked APIs are renamed from Xamarin SfMaps to maintain the consistency of API naming in the .NET MAUI SfMaps.
+
+The following code example explains how to intialize the zooming and it's customization in Xamarin SfMaps and .NET MAUI SfMaps.
```

Comment:
The following code example explains how to initialize the tile layer and its customization in Xamarin SfMaps and .NET MAUI SfMaps.